layouts/shortcodes/readfile.md

```diff
@@ -1,11 +1,53 @@
+{{/*
+
+Use this shortcode to include the contents a file into the location from
+where this shortcode is called. You can use %% (to send to markdown processor) or < >
+
+Syntax:
+    * For Markdown: Use %, {{%...%}} to send the file to the Markdown processor and enable in-page TOC.
+      Example:
+      * {{% readfile file="README.md" relative="true" %}}
+
+    * For HTML: Use < >, {{<...>}} to copy in the HTML as is.
+      Example:
+      * {{< readfile file="HTML.html" relative="true" >}}
+
+    * For code: Use < >, {{<...>}} See the code and lang parameters below for syntax highlighting.
+      Example:
+      * {{< readfile file="code-written-in.go" relative="true" code="true" lang="go" >}}
+
+Parameters:
+    * relative="true": Required if you specify a relative filepath in `file`. This is recommend for all content that is
```

Comment:
Can we get `relative` to be `true` by default?

If not, can we change this to be `absolute`? (So that the default is `absolute="false"`?

Reply:
yeah i like that idea. we are overriding an existing function so my initial reaction was to make it clear that we are changing the behavior but by making `relative` true by default will shorten and make this method easier to use (plus given our versioning, i dont think there is a scenario to ever use an absolute url)

Reply:
done